### PR TITLE
Tidy up repo before first release

### PR DESCRIPTION
**Pull request by p-j-smith**
- use the mirsg pre-commit and renovate configs
- use the mirsg `linting` and `molecule-test` actions
- remove the empty `handlers` directory
- use flat names for variables rather than nested

### CODE REVIEW

File: molecule/resources/converge.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
   pre_tasks:
```
Comment (drmatthews) [inline molecule/resources/converge.yml:7]: Unfortunately I left a mistake on line 18:

```
failed_when: systemctl_status.rc > 1
```

Should be:
```
failed_when: systemctl_status.rc > 0
```

**Review by drmatthews — CHANGES_REQUESTED** [review]
Many thanks for tidying up and getting this up-to-date @p-j-smith!

File: .github/workflows/add-issue-to-project.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      - opened
+
+jobs:
+  add-issue-to-project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: UCL-MIRSG/.github/actions/add-to-project@v0.24.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          app_pem: ${{ secrets.APP_PEM }}
```
Comment (paddyroddy) [inline .github/workflows/add-issue-to-project.yml]: This got changed yesterday UCL-MIRSG/.github#40
```suggestion
          app-id: ${{ secrets.APP_ID }}
          app-pem: ${{ secrets.APP_PEM }}
```

**Review by paddyroddy — CHANGES_REQUESTED** [review]
One action change

File: molecule/resources/converge.yml (hunk 1 above)
Comment (p-j-smith) [reply molecule/resources/converge.yml:7]: thanks for spotting that! I'll fix now

**Review by paddyroddy — APPROVED** [review]

**Review by drmatthews — APPROVED** [review]
👍

**Pull request merged**